src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -119,6 +119,18 @@
             </td>
             <td>{{ selection?.retain_head_object }}</td>
           </tr>
+          <tr>
+            <td
+                class="bold">
+              Allow Read Through
```

Comment:
```suggestion
              <span i18n >Allow Read Through </span>
```
Always use span when no direct warpper component for text